packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.spec.ts

```diff
@@ -64,19 +64,6 @@ describe('getProductBlockTitle()', () => {
         );
     });
 
-    it("returns '[title]...'  when title field is to long", () => {
```

Comment:
What is the new policy on titles being to long?

Reply:
They are truncated by EuiTextTruncate now. The full title is shown in the product block view.